assets/src/blocks/Gallery/getImageTitle.js

```diff
@@ -0,0 +1,6 @@
+export const getImageTitle = (image) => {
+  const caption = image.caption || '';
+  const credits = image.credits && !caption.includes(image.credits)
+    ? (image.credits.includes('©') ? image.credits : `© ${image.credits}`) : '';
```

Comment:
There's not many cases where a double ternary is readable, and I'm afraid this is not one of them :stuck_out_tongue:

Reply:
You could instead do
```js
const credits = !image.credits || caption.includes(image.credits) ? '' : `© ${image.credits.replace(/^©\s*)}`;

Reply:
Maybe an additional const helps too
```js
const dontAddCredits = !image.credits || caption.includes(image.credits);
const credits = dontAddCredits ? '' : `© ${image.credits.replace(/^©\s*)}`;